shell/platform/android/io/flutter/view/AccessibilityViewEmbedder.java

```diff
@@ -220,6 +225,113 @@ private void copyAccessibilityFields(AccessibilityNodeInfo input, AccessibilityN
         }
     }
 
+    /**
+     * Delegates an AccessibilityNodeProvider#requestSendAccessibilityEvent from the AccessibilityBridge to the embedded
+     * view.
```

Comment:
Can you include the meaning of the return value in the javadoc?

Reply:
done